src/ast/semantic_analyser.cpp

```diff
@@ -850,10 +851,28 @@ void SemanticAnalyser::visit(FieldAccess &acc)
   }
 }
 
-void SemanticAnalyser::visit(Cast &cast)
-{
+void SemanticAnalyser::visit(Cast &cast) {
```

Comment:
Please don't change the styling unless you're already changing the line and the new styling matches the majority of the code base (neither is the case here), otherwise these changes make reviewing the code harder as now there are unrelated lines changed in the file.

(eventually we'll have `clang-format` to enforce a consistent format across our files)